tck/running_the_tck.asciidoc

```diff
@@ -16,39 +16,91 @@
 
 = Running the Microprofile Config TCK
 
-Any Microprofile 1.2 and higher release must pass this test suite.
-The TCK uses `testng`
+Any Microprofile 2.0 and higher release must pass this test suite. The TCK uses `TestNG` and `Arquillian`.
```

Comment:
Any MicroProfile 1.2 means the umbrella spec. I don't think this should be changed.